build/testdata/bundles/terraform/porter.yaml

```diff
@@ -4,29 +4,66 @@ mixins:
 name: terraform
 version: 0.1.0
 registry: getporterci
+schemaVersion: 1.0.0
 
 parameters:
   - name: file_contents
     type: string
     applyTo:
       - install
       - upgrade
+  - name: map_var
```

Comment:
Might be a good idea to add defaults to verify/test yaml notation maintains quotes as expected.